manifesto/personas.md

```diff
@@ -0,0 +1,63 @@
+# Kyma Personas
+
+The following personas are fictional characters, which we defined in order to represent the different user types that might use Kyma. Keep in mind those personas when contributing to Kyma as they will help you to understand Kyma users' needs and goals (GAINS :green_heart:) and major pain points (PAINS :broken_heart:).
```

Comment:
```suggestion
The following personas are fictional characters that we defined in order to represent the different types of users that might use Kyma. Keep these personas in mind when contributing to Kyma as they will help you to understand Kyma users' needs and goals (GAINS :green_heart:) and major pain points (PAINS :broken_heart:).
```

Reply:
The construction is `keep sb/sb in mind`. See the [reference](https://idioms.thefreedictionary.com/keep+in+mind).